services/storage/tests/unit/test_dsm.py

```diff
@@ -215,18 +215,15 @@ async def test_copy_s3_s3(
 
 
 # NOTE: Below tests directly access the datcore platform, use with care!
+@pytest.mark.skipif(not has_datcore_tokens(), reason="no datcore tokens")
```

Comment:
Nice. How do you make sure that at least somewhere these tests run?

Reply:
as explained in the PR description above. these tests were already disabled before by default because there are no datcore token in our CI anyway. 
Before my changes, the test were falsely run: with a
```python
if not has_datcore_tokens():
  return
```
now it is a skipif which is more honnest I'd say :rofl:

Reply:
Make sense! 👍